options.css

```diff
@@ -34,6 +59,58 @@ button {
 	background-color: #292929;
 }
 
+/* Disable arrow buttons on the right side of input field.
+ * So that user could only redact text there, not to do it via mouse scroll.
+ */
+#z_index {
+	appearance: textfield;
+}
+
+#page_1 {
+	max-height: var( --max-page-height );
+}
+#page_2 {
+	/* Hide domains page by default */
+	display: none;
+	max-height: var( --max-page-height );
+}
+
+#domains_container {
+	overflow-y: scroll;
+	border: 1px;
+	min-height: var( --min-page-height );
+	max-height: var( --min-page-height );
+}
+.domains {
+	background-color: darkgrey;
+	border: 1px;
+	color: #000000;
+	font-family: monospace;
+	font-size: 16px;
+}
+.domain_element {
+	padding-top: 10px;
+	padding-left: 2px;
+}
+
+/* Default values for opening settings page */
+#page_1_button {
```

Comment:
The buttons now say "Page 1" and "Page 2". Maybe "General Settings" and "Excluded Domains"?

Reply:
What about "domains", yes, they certainly now only for "not showing clocks there". But some may add some more additional buttons for them, like: dark/light theme for it (and not hide, of course) (not done, and my bet, not in near future).

Reply:
Sounds good (capitalized: Domains).

But since the list is only **excluded** domains for now, please add explanatory text to say so.

Reply:
Also, "auto apply" doesn't work for me.

Reply:
You used "save button" for that? I transferred some changes from a pool of them, so i maybe missed some things.

Reply:
Oh, yes, `browser` is a firefox thing, chrome only understands `chrome`.